gems/smithy-client/sig/smithy-client/block_io.rbs

```diff
@@ -0,0 +1,13 @@
+module Smithy
+  module Client
+    class BlockIO
+
```

Comment:
What happens if you run rubocop against rbs files? Extra spacing in a few files.

Reply:
Can you run rubocop on rbs files? I wasn't able to get it working.

Reply:
I've never tried. That's what I was asking.